LekceDveSamples/LekceDveSamples/UserLoginTests.cs

```diff
@@ -0,0 +1,35 @@
+using FluentAssertions;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+
+namespace LekceDveSamples
+{
+    public class UserLoginTests
+    {
+        public WebDriver webDriver = new ChromeDriver();
+        string baseUrl = "";
+
+
+        [Test]
+        public void GivenLoginPageIsLoaded_WhenUserLoginWithValidCredentials_ThenIsLoggedIn()
+        {
+            webDriver.Navigate().GoToUrl(baseUrl);
+            webDriver.FindElement(By.Id("LoginField")).SendKeys("Ucastnice");
+            webDriver.FindElement(By.Id("PasswordField")).SendKeys("Heslo");
+            webDriver.FindElement(By.Id("OkButton")).Click();
+            webDriver.FindElement(By.Id("WelcomeBanner")).Text
+                .Should().Be("Prihlaseni problehlo uspesne");
```

Comment:
"Prihlaseni probehlo uspesne" -> const nebo tak něco, už je to ve fajlu 2x a navíc je předpoklad že to bude často používaný text

Reply:
Ted si budu trochu protirecit :D
tady bych to neresil. V lekci dva kdy toho je malo bych to tak nechal a pozdeji by to bylo podle me validnejsi kdyz uz clovek bude psat vice testu za lekci.
Vim ze jsem zrovna napsal u namingu toho testu s uzivatelem ze "Valid" bych tam nechal, protoze pak jsou jeste "invalid" a "valid with invalid" :D ale na druhou stranu tahle lekce uz se zameruje na naming.

Jak chces, zvaz to jeste, ja bych to pro prehlednost a jednoduchost nechal as is.